ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -288,7 +288,7 @@ Spawn Notebook With Arguments  # robocop: disable
                 END
             END
             Spawn Notebook    ${spawner_timeout}    ${same_tab}
-            Run Keyword And Continue On Failure    Wait Until Page Contains    Log in with OpenShift    timeout=15s
+            Run Keyword And Warn On Failure    Wait Until Page Contains    Log in with OpenShift    timeout=15s
```

Comment:
In a single user session it will only appear the first time the user tries to access a notebook/workbench, and after that it's cached. 
The wait is still important otherwise the following keyword could miss the actual oauth login page, but we don't want to raise a failure in case it does not appear due to caching